corehq/util/tests/test_couch.py

```diff
@@ -253,7 +253,7 @@ def test_a_few_important_ones(self):
             self.assertEqual(model_class, get_document_class_by_name(model_name))
 
     def test_missing(self):
-        test_cases = [None, '', 'FooDocument']
+        test_cases = [None, 'FooDocument']
```

Comment:
As to *why* it changed, my guess is it's because nose runs tests in a different order than Django runs them. However, I have no idea why it's an *intermittent* failure with nose.